pyega3/pyega3.py

```diff
@@ -568,9 +579,18 @@ def main():
     if args.debug:
         global logging_level
         logging_level = logging.DEBUG
-        print("[debugging]")
 
-    logging.basicConfig(level=logging_level, format='%(asctime)s %(message)s', datefmt='[%Y-%m-%d %H:%M:%S %z]')
+    logging.basicConfig(level=logging_level, format='%(asctime)s %(message)s', datefmt='[%Y-%m-%d %H:%M:%S %z]',  
+    					handlers=[
+            logging.FileHandler("pyega3_output.log"),
+            logging.StreamHandler()
+        ])
+
+    logging.info("")
```

Comment:
Is this for insert a blank line?

Reply:
For every command adds a blank line in the start which will separate the new command logs from previous commands logs